vertx-web/src/main/java/io/vertx/ext/web/impl/ParsableMIMEValue.java

```diff
@@ -27,6 +31,25 @@ public String subComponent() {
     return subComponent;
   }
 
+  @Override
+  public String mediaType() {
+    ensureHeaderProcessed();
+    return component + "/" + subComponent;
+  }
+
+  @Override
+  public String mediaTypeWithParams() {
+    ensureHeaderProcessed();
+    StringBuilder sb = new StringBuilder(component + "/" + subComponent);
+    if (!parameters().isEmpty()) {
+      sb.append("; ");
+      sb.append(parameters().entrySet().stream().map(ParsableMIMEValue::encodeParam).collect(Collectors.joining("; ")));
```

Comment:
We already have a `StringBuilder` object, instead of creating a stream we can iterate over parameters and append encoded params to it.

Reply:
Also, the `parameters()` value can be stored in a local variable